src/data_provider/tests/sysInfoProcessesSolaris/sysInfoSolarisProcesses_test.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * Wazuh SysInfo
+ * Copyright (C) 2015, Wazuh Inc.
+ * March 10, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include <fstream>
+#include <unistd.h>
+
+#include "sysInfoSolarisProcesses_test.h"
+#include "processes/processSolarisWrapper.hpp"
+#include "processes/processFamilyDataFactory.h"
+
+void SysInfoSolarisProcessesTest::SetUp(){};
+void SysInfoSolarisProcessesTest::TearDown(){};
+
+using ::testing::_;
+using ::testing::Return;
+
+TEST_F(SysInfoSolarisProcessesTest, Test_Success_Data_1)
+{
+    nlohmann::json Processes{};
+
+    // the injected data was extracted from a real solaris terminal
+    std::ifstream ifInfo{"psinfo.985", std::ios::binary};
+    std::ifstream ifStatus{"status.985", std::ios::binary};
+    std::ifstream ifCred{"cred.985", std::ios::binary};
```

Comment:
because the data is binary in solaris, it is difficult to emulate the real conditions, also the mocking is irrelevant and does not contribute to testing, the premise was to extract and parse